buildSrc/src/main/kotlin/codegen/GenerateKotlinDependencyExtensions.kt

```diff
@@ -67,7 +64,7 @@ val embeddedKotlinVersion = "$embeddedKotlinVersion"
  */
 @Deprecated("Will be removed in 1.0")
 fun RepositoryHandler.gradleScriptKotlin(): ArtifactRepository =
-    maven { it.setUrl("$kotlinDslRepository") }
+    gradlePluginPortal()
```

Comment:
❓Could you explain why this change?

Reply:
`kotlinDslRepository` was always the constant `"https://plugins.gradle.org/m2/"` which got removed in 027141713ee6470d768eda980b36b81d6a5ed449 because unnecessary.